components/front-page/PopularCategories.tsx

```diff
@@ -28,10 +28,10 @@ const Category = ({
 }) => {
   return (
```

Comment:
I can't seem to be able to scroll through horizontally the popular categories anymore. (tested on desktop only)

![Screenshot 2023-09-05 at 9 47 24 PM](https://github.com/zeitgeistpm/ui/assets/14626415/1e8865ab-2f91-450f-b6cc-6af9593493cc)

Reply:
Touch events(dragging) seems to work tough so dont thin this will be an issue on mobile.

Reply:
Okay, sounds good